README.md

```diff
@@ -1,34 +1,56 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
 
-## Our target audience
-Developers for now (hence the name), but we welcome anyone that wants to join the community.
+## Who we are
+Our membership is primarily made up of people whose roles are involved in the tech industry and our partnerships include those whose livelihoods and businesses depend on it. Anyone involved directly or indirectly with tech of any kind or those who care about the tech community are invited to join our ranks.
```

Comment:
There are two distinct things going on here. I tried like 10 different things here, but none of them were really acceptable. It's difficult to describe two distinct sets of people in the same sentence. I'm giving this a day or so before returning to it.